crt_portal/cts_forms/model_variables.py

```diff
@@ -153,17 +153,22 @@
     ('VOT', _('Voting')),
 )
 
-PLACE_CHOICES = (
-    ('home', _('Home, potential home, or services to help with purchasing a home (banks, lenders, or other financial services)')),
-    ('workplace', _('Workplace or potential workplace')),
-    ('school', _('Educational institution (school, university), education program or educational activity (after school program or workshop)')),
+COMMERCIAL_OR_PUBLIC_PLACE_CHOICES = (
     ('place_of_worship', _('Place of worship')),
```

Comment:
[content doc](https://docs.google.com/document/d/1bxSRc9CQOHjQhTaYkHncE4tykUEFeC6RzFtP5IQk0Ic/edit?ts=5de6f69e#) says "Place of worship or about a place of worship"